### PR TITLE
Add flake8 to pre commit

### PR DESCRIPTION
**Pull request by jgersak**

### CODE REVIEW

File: land-cover-fastai/deep_lulc_walkthrough.ipynb [hunk 1]
```diff
@@ -168,7 +176,7 @@
     "    patchID = np.array(range(len(tiles))).astype(\"int\")\n",
     "    geometry = [Polygon(bbox_.get_polygon()) for bbox_ in tiles[patchID]]\n",
     "\n",
-    "    while save == True:\n",
+    "    while save is True:\n",
```
Comment (zigaLuksic) [inline land-cover-fastai/deep_lulc_walkthrough.ipynb:179]:
```suggestion
    "    while save:\n",
```

File: land-cover-fastai/deep_lulc_walkthrough.ipynb [hunk 2]
```diff
@@ -209,7 +217,7 @@
     "    patch_data = EOPatch.load(patch)\n",
     "    bands = patch_data.data[\"FEATURES\"]\n",
     "    patch_dir, patch_id = patch.split(\"/\")\n",
-    "    if inference == True:\n",
+    "    if inference is True:\n",
```
Comment (zigaLuksic) [inline land-cover-fastai/deep_lulc_walkthrough.ipynb:220]:
```suggestion
    "    if inference:\n",
```

File: land-cover-fastai/deep_lulc_walkthrough.ipynb [hunk 3]
```diff
@@ -553,26 +566,26 @@
...
+    "    # name2id = {v: k for k, v in enumerate(codes)}\n",
+    "    # void_code = name2id[\"NoData\"]\n",
     "\n",
     "    metrics = iou\n",
     "    wd = 1e-2\n",
     "\n",
     "    learner = unet_learner(data, models.resnet50, metrics=metrics, wd=wd)\n",
     "    lr = 1e-4\n",
-    "    if train_bol == False:\n",
+    "    if train_bol is False:\n",
```
Comment (zigaLuksic) [inline land-cover-fastai/deep_lulc_walkthrough.ipynb:577]:
```suggestion
    "    if not train_bol:\n",
```

File: land-cover-fastai/deep_lulc_walkthrough.ipynb [hunk 4]
```diff
@@ -665,7 +678,7 @@
     "    pred_path = \"pred_inference\"\n",
     "    if not op.isdir(pred_path):\n",
     "        makedirs(pred_path)\n",
-    "    if inference == True:\n",
+    "    if inference is True:\n",
```
Comment (zigaLuksic) [inline land-cover-fastai/deep_lulc_walkthrough.ipynb:681]:
```suggestion
    "    if inference:\n",
```

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 5]
```diff
@@ -454,11 +441,11 @@
     "        tile_splitter = BBoxSplitter([aoi_shape], t_crs, (width_grid, heigth_grid))\n",
     "\n",
     "        tile_list = np.array(tile_splitter.get_bbox_list())\n",
-    "        info_list = np.array(tile_splitter.get_info_list())\n",
+    "        # info_list = np.array(tile_splitter.get_info_list())\n",
```
Comment (zigaLuksic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:444]: why is this commented out?

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 6]
```diff
@@ -2370,18 +2343,16 @@
...
     "from os import path as op\n",
     "\n",
     "import geopandas as gpd\n",
     "import keras.backend as K\n",
     "import numpy as np\n",
     "import pandas as pd\n",
-    "import yaml\n",
-    "from keras.applications.vgg16 import VGG16\n",
+    "\n",
+    "# import yaml\n",
```
Comment (zigaLuksic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2354]: why is this commented out?

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 7]
```diff
@@ -2564,10 +2534,11 @@
...
     "\n",
-    "    # Load model parameters for printing\n",
-    "    with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
-    "        params_yaml = f_model_params.read()\n",
-    "        model_params = yaml.load(params_yaml)\n",
+    "    # # Load model parameters for printing\n",
+    "    # with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
+    "    #     params_yaml = f_model_params.read()\n",
+    "    #     model_params = yaml.load(params_yaml)\n",
+    "\n",
```
Comment (zigaLuksic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2541]: ?

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 8]
```diff
@@ -2790,9 +2761,9 @@
     "    )\n",
     "\n",
     "    all_preds = list()\n",
-    "    with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
-    "        params_yaml = f_model_params.read()\n",
-    "        model_params = yaml.load(params_yaml)\n",
+    "    # with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
+    "    #     params_yaml = f_model_params.read()\n",
+    "    #     model_params = yaml.load(params_yaml)\n",
```
Comment (zigaLuksic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2766]: ?

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 9]
```diff
@@ -2815,8 +2786,8 @@
     "        ###############################\n",
     "        # Predict on batches of images\n",
     "        ###############################\n",
-    "        y_preds = []\n",
-    "        pred_dict = dict()\n",
+    "        # y_preds = []\n",
+    "        # pred_dict = dict()\n",
```
Comment (zigaLuksic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2790]: ?

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb (hunk 5 above)
Comment (jgersak) [reply poverty-detection-keras/Poverty_Detection_Workflow.ipynb:444]: never used

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb (hunk 6 above)
Comment (jgersak) [reply poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2354]: import for never used

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb (hunk 7 above)
Comment (jgersak) [reply poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2541]: never used

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb (hunk 8 above)
Comment (jgersak) [reply poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2766]: never used

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb (hunk 9 above)
Comment (jgersak) [reply poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2790]: never used

**Pull request merged**